winui/Rating/Customization.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Customization with WinUI Rating control | Syncfusion
+description: Learn here all about UI customization with Syncfusion WinUI Rating Control (SfRating), its elements, and more.
```

Comment:
description: Learn about UI customization with Syncfusion WinUI Rating Control (SfRating), its elements, and more here.

Reply:
Changed the description as mentioned above.